banzai/qc/header_checker.py

```
@@ -108,16 +117,21 @@ def check_header_na(self, keyword, image):
                 a  banzais.image.Image object.
 
         """
-
-        header_value = image.header[keyword]
-
         logging_tags = logs.image_config_to_tags(image, self.group_by_keywords)
-
-        if isinstance(header_value, str):
-
-            if 'N/A' in header_value:
-                sentence = 'The header key ' + keyword + ' got the unexpected value : N/A'
-                self.logger.error(sentence, extra=logging_tags)
+        na_keywords = []
+        for keyword in keywords:
+            try:
+                header_value = image.header[keyword]
+                if isinstance(header_value, str):
+                    if 'N/A' in header_value:
+                        sentence = 'The header key ' + keyword + ' got the unexpected value : N/A'
+                        self.logger.error(sentence, extra=logging_tags)
+                        na_keywords.append(keyword)
+            except:
```

Should this really swallow all exceptions? Or just `KeyError`?